diff --git a/default b/default
index c5af914..a211612 100644
--- a/default
+++ b/default
@@ -38,18 +38,19 @@ server {
 	#
 	# include snippets/snakeoil.conf;
 
-	root /var/www/html;
+	#root /var/www/html;
 
 	# Add index.php to the list if you are using PHP
-	index index.html index.htm index.nginx-debian.html;
+	#index index.html index.htm index.nginx-debian.html;
 
 	server_name _;
+	return 301 https://$server_name$request_uri;
 
-	location / {
+	#location / {
 		# First attempt to serve request as file, then
 		# as directory, then fall back to displaying a 404.
-		try_files $uri $uri/ =404;
-	}
+	#	try_files $uri $uri/ =404;
+	#}
 
 	# pass PHP scripts to FastCGI server
 	#
